modules/gitops-argo-rolouts-cli-ovr.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argo-rollouts-cli-overview_{context}"]
+= Argo Rollouts CLI Overview
+
+Using the Argo Rollouts CLI plugin is optional because you can deploy and monitor an application by installing Argo Rollouts using the {gitops-title} Operator. Following the installation of Argo Rollouts, users have the choice to utilize the Argo Rollouts CLI plugin. This plugin effectively handles and displays Argo Rollouts resources directly from the command line interface.
```

Comment:
>  Using the Argo Rollouts CLI plugin is optional because you can deploy and monitor an application by installing Argo Rollouts using the {gitops-title} Operator. Following the installation of Argo Rollouts, users have the choice to utilize the Argo Rollouts CLI plugin. This plugin effectively handles and displays Argo Rollouts resources directly from the command line interface.

What about:

The Argo Rollouts CLI plugin is not required to use Argo Rollouts, because you can instead deploy and monitor a Rollout resource using just the OpenShift Console or oc command. However, this plugin effectively handles and displays Argo Rollouts resources directly from the command line interface.

(The original text made it sound like the CLI plugin was involved in installing/monitoring the Argo Rollouts installation itself, which it is not.)

Reply:
Fixed.